serverlessFunctions/transit/transit.js

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017 HERE Europe B.V.
+ * Copyright (c) 2019 HERE Europe B.V.
```

Comment:
Don't just replace the year. If this needs to be updated, use a range like in `Copyright (c) 2017-2019 HERE Europe B.V.`.